dags/hivemind_website_ingestion.py

```
@@ -0,0 +1,98 @@
+        process_discord_guild_mongo(
+            community_id=community_id,
+            platform_id=platform_id,
+            selected_channels=selected_channels,
+            default_from_date=from_date,
+        )
+        logging.info(
+            f"Community {community_id} Job finished | platform_id: {platform_id}"
+        )
+
+    communities_info = get_website_communities()
+    start_discord_vectorstore.expand(community_info=communities_info)
```

Ruff

Undefined name `start_discord_vectorstore`

(F821)